apps/challenges/challenge_config_utils.py

```diff
@@ -112,14 +135,11 @@ def is_challenge_phase_config_yaml_html_field_valid(yaml_file_data, key, base_lo
     message = ""
     is_valid = False
     if value:
-        file_path = join(base_location, value)
-        if file_path.endswith(".html") and isfile(file_path):
-            is_valid = True
-        else:
-            message = (" ERROR: There is no html file present for key"
-                       " {} in phase {}.".format(key, yaml_file_data["name"]))
+        is_valid = True
```

Comment:
@KhalidRmb why is this check removed?

Reply:
Because the validation inside the challenge create API is such that it allows an invalid value for the file. It just replaces it with null before moving on. Validation fails if the entire key is missing. @Ram81

Reply:
I didn't get the reason for the removal of this check. Can we please discuss about this on slack channel.